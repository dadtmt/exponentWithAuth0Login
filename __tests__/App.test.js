import React from 'react'
import renderer from 'react-test-renderer'

import App from '../App'

it('renders content', () => {
  expect(
    renderer.create(<App />).toJSON()
  ).toMatchSnapshot()
})
