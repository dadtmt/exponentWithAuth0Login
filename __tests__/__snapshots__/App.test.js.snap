exports[`test renders content 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}>
    <View
      style={
        Array [
          Object {},
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#0C42FD",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }>
        Login with Auth0
      </Text>
    </View>
  </View>
</View>
`;
